game/src/main/java/net/demilich/metastone/game/cards/Card.java

```diff
@@ -894,6 +894,13 @@ public boolean canBeCast(GameContext context, Player player) {
 		}
 
 		Player opponent = context.getOpponent(player);
+
+		if (getCondition() != null) {
```

Comment:
this is definitely a little mysterious of a change but i believe you, what a catch